Packs/CortexXDR/Playbooks/Cortex_XDR_-_Isolate_Endpoint6_2.yml

```diff
@@ -21,6 +21,7 @@ tasks:
       '#none#':
       - "2"
```

Comment:
Change the file names to be with the minimum supported version ( add 6.2 for the new ones).